tests/install_current_branch.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+
+# Use this script to install lightly at the current branch
```

Comment:
what does it mean to install "lightly at the current branch"?
- latest master?
- local branch?

Reply:
deleted this file